core/templates/pages/topics-and-skills-dashboard-page/create-new-topic-modal.controller.spec.ts

```diff
@@ -129,4 +129,8 @@ describe('Create new topic modal', function() {
         TopicEditorStateService.updateExistenceOfTopicName
       ).not.toHaveBeenCalled();
     });
+
+  it ('should update View when thumbnail has been uploaded', function() {
+    $scope.updateView();
+  });
```

Comment:
@srijanreddy98 Is this the right way to hit this line ? The function can't be called anywhere else